resources/views/frontend/layouts/partials/menu.blade.php

```diff
@@ -0,0 +1,155 @@
+<div class="body body2">
+  <div id="page">
+    <div id="header" class="headerwrap">
+      <div class="headert">
+        <div class="hd_bg_">
+          <script type="text/javascript">
+            (function($){
+              $(function(){
+                $('.parallax-layer', '#parallax').parallax({
+                  mouseport: $('body')}
+                                                          );
+              }
+               );
+            }
+            )(jQuery);
+          </script>
+          <div class="parallax-viewport" id="parallax">
+            <!-- parallax layers -->
+            <div class="parallax-layer" class="layer1" >
+              <img src="{{asset('{{asset('frontend/imgs/hd_bg.jpg')}}" alt=""  id="img1" style=";" />
+            </div>
+            <div class="parallax-layer" class="layer2" >
+              <img src="{{asset('frontend/imgs/hd_may1.png')}}" alt="" id="img2"  />
+            </div>
+            <div class="parallax-layer"  class="layer3" >
+              <img src="{{asset('frontend/imgs/hd_may2.png')}}" alt=""  id="img3" style=";" />
+            </div>
+            <div class="parallax-layer" class="layer4" >
+              <img src="{{asset('frontend/imgs/hd_may3.png')}}" alt="" id="img4"  />
+            </div>
+            <div class="parallax-layer" class="layer5" >
+              <img src="{{asset('frontend/imgs/onlinemath.png')}}" alt="" id="img5" />
+            </div>
+            <div class="parallax-layer" class="layer6" >
+              <img src="{{asset('frontend/imgs/5_bg_chamhoc_slg.png')}}" alt="" id="img6"  />
+            </div>
+            <div class="parallax-layer" class="layer7" >
+              <img src="{{asset('frontend/imgs/hd_tng.png')}}" alt="" id="img7"  />
+            </div>
+            <div class="parallax-layer" class="layer8" >
+              <img src="{{asset('frontend/imgs/hd_tng1.png')}}" alt="" id="img8"  />
+            </div>
+            <div class="parallax-layer" class="layer9" >
+              <img src="{{asset('frontend/imgs/hd_dayleo.png')}}" alt="" id="img9"  />
+            </div>
+          </div>
+        </div>
+        <div class="hd_content">
+          <a class="logo" href="">
+          </a>
+        </div>
+      </div>
+      <div id="main-menu">
+        <div class="nav_bar">
+          <div class="content" style="float: none;">
+            <ul id="menu" class="nav">
+              <li class="trangchu trangchu2" style="display: block; position: relative;">
```

Comment:
[rank A] Again. You are using css inline. Please pay attention for these mistake, you are getting it so many times.